frontend/src/components/pages/search/Search.tsx

```diff
@@ -67,6 +67,8 @@ export const SearchUI: React.FC<Props> = ({ language }) => {
     resetTextFilter,
   } = useTextFilter();
 
+  const [dateFilter, setDateFilter] = useState({ beginDate: '', endDate: '' });
```

Comment:
In reaction with 
> Hello @dtrucs :)
> 
> For this PR, we've made progress but we still have a problem. The URL is updating when the date filter has a value. However, we are unable to pre-fill the date fields when the url is copied and pasted. We looked in the code but couldn't find where this is done for others filters. Thanks a lot for ur help

This line, whatever the url contains you define empty strings.
Take a look of the `useTextFilter` hook two lines above. Maybe you can refacto/rename the hook to contain `dateFilter` values.